source/content/guides/drupal/drupal-hosted-md/07-drupal-core-version.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
@stovak , still contains multiple references to D9.  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-hosted-md/drupal-core-version

Reply:
same